physical/cassandra.go

```diff
@@ -86,6 +122,71 @@ func newCassandraBackend(conf map[string]string, logger log.Logger) (Backend, er
 	return impl, nil
 }
 
+func setupCassandraTLS(conf map[string]string, cluster *gocql.ClusterConfig) error {
+	tlsOnStr, ok := conf["tls"]
+	if !ok {
+		return nil
+	}
+
+	tlsOn, err := strconv.Atoi(tlsOnStr)
+	if err != nil {
+		return fmt.Errorf("'tls' must be an integer (0 or 1)")
```

Comment:
It's a bit confusing. I'll dig deeper but while trying unquoted literals in both config formats I had this same issue: https://github.com/hashicorp/vault/issues/1559